ios/Podfile.lock

```diff
@@ -83,10 +83,12 @@ PODS:
     - React-Core (= 0.60.5)
   - ReactNativeLocalization (2.1.5):
     - React
-  - RNCAsyncStorage (1.6.1):
+  - RNCAsyncStorage (1.5.1):
```

Comment:
Why did you downgrade this library?

Reply:
That's because I locked the versions, will upgrade it now, good catch